static/templates/user-groups-admin.handlebars

```diff
@@ -1,5 +1,8 @@
 <div id="user-groups-admin" class="settings-section" data-name="user-groups-admin">
     <div class="tip">Anyone in this organization can add user groups.</div>
+    {{#unless is_admin}}
+    <div class="tip">Only user group members can modify their respective group name/description/members.</div>
```

Comment:
Is't this better?
`Only user group members and organization administrators can modify the group.`

Reply:
I think showing `Only organization administrators can edit these settings` here doesn't make any sense. What do you think?
![screenshot from 2018-02-27 01-47-53](https://user-images.githubusercontent.com/7190633/36693128-4c3b10ee-1b60-11e8-9aa7-946e0c1773d9.png)

Reply:
There is no setting for disabling users from adding new groups right?. In that case we can delete this box as well.
`Anyone in this organization can add user groups.` Have a separate commit for this alert box changes including the one I suggested before.

Reply:
@hackerkid thanks for this detailed review, I will surely update these as soon as I can.
I agree with you on the last two points regarding the removal of boxes, they don't make sense in the current scenario.